lib/config/guiconfiguration.go

```diff
@@ -9,12 +9,14 @@ package config
 import (
 	"net/url"
 	"os"
+	"strconv"
 	"strings"
 )
 
 type GUIConfiguration struct {
 	Enabled                   bool     `xml:"enabled,attr" json:"enabled" default:"true"`
 	RawAddress                string   `xml:"address" json:"address" default:"127.0.0.1:8384"`
+	UnixSocketPermissions     string   `xml:"unixSocketPermissions,omitempty" json:"unixSocketPermissions"`
```

Comment:
The usual pattern for fields that shouldn't be accessed directly is prepending `Raw`, d.h. field is `RawUnixSocketPermissions` and the method below `UnixSocketPermissions`.

Reply:
Done.